### PR TITLE
add tags and details

### PR DESCRIPTION
**Pull request by akshaya-a**

### CODE REVIEW

File: examples/AzureML-REST-API/submit-custom-image.md [hunk 1]
```diff
@@ -78,12 +78,49 @@ resp = requests.post(start_run, files=run_files, headers=header)
...
+```
+
+## Retrieve a run by tag
+
+Later, you can fetch the run by querying the experiment
+
+```python
+query_runs = hosturl + historybase + resourcebase + "experiments/{}/runs:query".format(experiment_name)
+payload = {
+   "filter": "Tags/customimage eq specialrun",
```
Comment (jpe316) [inline examples/AzureML-REST-API/submit-custom-image.md:110]: How does this filtration syntax work exactly? is it documented?

File: examples/AzureML-REST-API/submit-custom-image.md [hunk 2]
```diff
@@ -78,12 +78,49 @@ resp = requests.post(start_run, files=run_files, headers=header)
...
+print(resp.json()["value"][0])
+```
+
 ## Monitor status of run
 
 You can track the status of the run by polling the run history service.
 
 ```python
-run_id = json.loads(resp.text)["runId"]
+run_id = resp.json()["value"][0]["runId"]
```
Comment (jpe316) [inline examples/AzureML-REST-API/submit-custom-image.md:123]: Would be good to show an example response

File: examples/AzureML-REST-API/submit-custom-image.md (hunk 1 above)
Comment (akshaya-a) [reply examples/AzureML-REST-API/submit-custom-image.md:110]: It's Odata with a slight tweak for nested properties. You're right that it's probably not documented sufficiently somewhere. We should do that eventually

File: examples/AzureML-REST-API/submit-custom-image.md (hunk 2 above)
Comment (akshaya-a) [reply examples/AzureML-REST-API/submit-custom-image.md:123]: What would you like to see here? The result of this is a RunDto which they would get from the previous walkthroughs, and the upperlevel response here is a paginatedresult, which is a list of RunDto effectively. Happy to add examples of either if you think they're useful

Comment (jpe316) [reply examples/AzureML-REST-API/submit-custom-image.md:123]: I was thinking just show an an example of a status returned

**Review by jpe316 — APPROVED** [review]

**Review by rastala — APPROVED** [review]

**Pull request merged**